src/utils/tokens.ts

```diff
@@ -139,27 +148,37 @@ const overrideCoingeckoId = (tokenInfo: TokenInfo): TokenInfo => {
           },
         };
         break;
-    }
-  } else {
-    const tags = tokenInfo.tags ?? [];
-    if (tags.includes("saber-mkt-luna")) {
-      updatedTokenInfo = {
-        ...tokenInfo,
-        extensions: {
-          ...tokenInfo.extensions,
-          coingeckoId: "terra-luna",
-        },
-      };
-    } else if (tags.includes("saber-mkt-sol")) {
-      updatedTokenInfo = {
-        ...tokenInfo,
-        extensions: {
-          ...tokenInfo.extensions,
-          coingeckoId: "solana",
-        },
-      };
+      case "wrapped-terra":
```

Comment:
Why does this have to be overridden? `wrapped-terra` has an API ID on CoinGecko

Reply:
I guess we are overriding `wrapped-bitcoin` too, just noticed. But still curious

Reply:
It was uncertain to me that these existed and I saw other cases where things in saber registry didn't exist, such as the `multi-collateral-dai` ID. How did you confirm `wrapped-terra` actually maps to something?

Reply:
Sorry, missed responding to this, wrapped-terra has an API ID here: https://www.coingecko.com/en/coins/wrapped-terra

I checked the API ID section too, it is `wrapped-terra`.

Reply:
Strange thing was we weren't getting prices for some LUNA assets that were using that ID. So now we enforce it to https://www.coingecko.com/en/coins/terra-luna which definitely works.

Reply:
Sounds good, will resolve this.